app/views/producer_mailer/order_cycle_report.text.haml

```diff
@@ -30,7 +30,7 @@ Orders summary
 = t :producer_mail_text_after
 
 #{t :producer_mail_signoff},
-#{@coordinator.name}
-#{@coordinator.address.address1}, #{@coordinator.address.city}, #{@coordinator.address.zipcode}
+#{raw(@coordinator.name)}
+#{raw(@coordinator.address.address1)}, #{@coordinator.address.city}, #{@coordinator.address.zipcode}
```

Comment:
I don't know, but it seems possible that the city, and maybe even zipcode would have special characters in them too.